src/resources/js/admin/help-page.js

```diff
@@ -84,10 +84,30 @@ tribe.helpPage = tribe.helpPage || {};
 			ajaxurl,
 			request,
 			function ( results ) {
+
 				if ( results.success ) {
 					obj.$system_info_opt_in_msg.html( "<p class='optin-success'>" + results.data + "</p>" );
 				} else {
-					obj.$system_info_opt_in_msg.html( "<p class='optin-fail'>" + results.data.message + " Code:" + results.data.code + " Status:" + results.data.data.status + "</p>" ); // eslint-disable-line max-len
+					var html = "<p class='optin-fail'>Something has gone wrong!</p>";
```

Comment:
Do we want to move out from var now that we allow es6?

Reply:
Probably. My linter was pestering me about it so I just changed it.

Reply:
I'm going to leave this one for now - as it follows the convention in the rest of the file.

Reply:
`Something has gone wrong!` needs to be translatable. I would suggest registering that string and the strings `Code:` and `Status:` on lines 101 and 105 get translated and made available in a JS object so this object can use those translated values.